econml/dml.py

```diff
@@ -236,6 +239,8 @@ def _combine(self, X, T, fitting=True):
                     else:
                         F = X
                 else:
+                    if not fit_cate_intercept:
```

Comment:
I think that if you need `fit_cate_intercept` here, you should store it as an attribute on this model during init.  As I recall, things that are captured from the outside context like this don't get handled correctly when the model is cloned and run on a different process when used with bootstrap.